### DISCUSSION TITLE
NDC at 8 digit level cost reporting

### DISCUSSION THREAD

**Discussion by atw1223a** [CMSgov/price-transparency-guide#91] [Q&A] [answered]
The prescription drug schema indicates that NDC reporting for prescription drugs should be done at the 8 digit level as the last two digits of an NDC are for package size/quantity. In looking into this further, for example, NDC 000740124 for Humira. The first 8 digits of the NDC are the same for Humira Pen Peds, Humira Pen Kit, Humira Pen 80/0.8ml, Humira Pen Kit CD/UC/HS. While each product has the same first 8 digits, the prices differ between the products. How do we determine which drug to select when each NDC is slightly different in price at the 10 digit level?

Another example is Cosentyx. The NDC 000780639 is used for both the 300 dose pen, 150 mg/ml pen, the 150 mg/ml pen, and the 300 dose. The price differs significantly depending on which full 10 digit NDC is used.

**Comment by dwdw1** [comment]
I'd also like to note that NDC 8 doesn't work in all instances therefore the format should be either NDC 9 or NDC 11 which are more widely used industry standards. NDC 8 doesn't work for products like Harvoni which lack a leading zero in the first two segments of the NDC and therefore cannot be made into NDC 8.
61958180301	HARVONI TAB 45-200MG
61958180101	HARVONI TAB 90-400MG
61958180501	HARVONI PAK   
61958180401	HARVONI PAK 45-200MG

NDC 8 would result in a ~55% reduction in record count, but this is achieved by omitting the 15% of NDC records which cannot be made into NDC 8. Meanwhile NDC 9 would result in a ~54% reduction in record count and can be done for all NDCs. Moving to NDC 9 makes much more sense, can be done for all NDCs, and results in a comparable record reduction.

However, I agree the price of a product can vary at the NDC 9 as @atw1223a points out. Therefore the most accurate representation would be NDC 11.

**Reply by nmayle** [reply]
It also isn't clear to me what the specification for this field means given that it says "A unique 10-digit or 11-digit, 3-segment number assigned by the FDA" and then it says "Data reporting will be on the first 8 digits of the full 10-digit or 11-digit NDCs."  Is the spec for an 8 digit NDC or a 10 or 11 digit code.  If it is for 11 digits then which one would be reported if there are multiple for the same 8 digit product code.

Summarizing various comments:
 @BobSyracuse points out, the rules says 10 or 11 digits
 @dwdw1 points out 8 digits doesn't even work for some products since there isn't a leading zero that could be elided
 @atw1223a points out that the price can vary across packages, so 10 or 11 digits are required

Also note this language in the Federal Register page 72222 (bold added):

> The final rules include a modification that clarifies that, in the case of prescription drugs, plans and issuers may only use the NDC as the billing code type because, as discussed later in this preamble, the accuracy of pricing information for prescription drugs requires precise and specific product information, **including package size** and manufacturer, which can only be achieved through the use of the NDC billing code. 

Where did the idea for 8 digits come from anyway?

**Comment by ghost** [comment]
In information provided, the regulation defines NDC on page 235 of CMS-9915-F as, 
_"a National Drug Code (NDC) (The final rules define the NDC code as a unique 10-digit or 11-digit 3-segment number assigned by the Food and Drug Administration (FDA), which provides a universal product identifier for drugs in the United States),168"_

So per @atw1223a 's question, why would the Prescription Drug schema have NCD at the 8 digit level?

**Comment by shaselton-usds** [comment, marked as answer]
With recent guidance dropping -- CMS will defer enforcement for the prescription drug files. You can find that information here: https://www.cms.gov/CCIIO/Resources/Fact-Sheets-and-FAQs/Downloads/FAQs-Part-49.pdf

**Comment by nmayle** [comment]
When the new prescription drug file schema is designed, please keep in mind the need for all of the digits of the NDC code since prices do vary by package and without the specific package it will not be clear what quantity of the drug the negotiated_rate corresponds to.

**Reply by shaselton-usds** [reply]
Thanks for the feedback.

**Comment by AH-GJ** [comment]
Hopefully the CMS guidance will clarify usage of the "outer" NDC rather than the "inner" NDC as well. When billing to Medicaids, the outer 11 digit code is always used. The FDA schema of removing "unnecessary" leading zeros from one of the 3 "segments" to achieve a 10 digit code has always created lots of confusion for me.

**Reply by nmayle** [reply]
I agree that while it isn't very hard for consumers of the data to write code to standardize the format of the NDCs, the file schema should specify how NDC codes are represented both in terms of the leading zeroes in the segments and whether dashes are allowed or not.

**Reply by dwdw1** [reply]
I hope the field is 11 digits string, no dashes but all 11 digits of the NDC. That's how nearly all systems and reporting I've seen at the NDC level operate. Adding dashes makes things messy and cumbersome.

**Reply by nmayle** [reply]
I agree.  The one downside to more specificity (11 digits versus less) is that given the out-of-network-like definition of the time window and volume threshold, we may get fewer codes/drugs making it through the hurdle.  That is if the thresholding is interpreted as applying to the specific packaging versus the broader drug.  One could argue that the packaging digits are like the modifiers on CPT and HCPCS and shouldn't be considered when doing the counting of volume.

The design of the out of network requirement certainly didn't result is much data. 

While a longer time window would make the prices less current and fuzzier, it would be one way to get more data.
